ruffd-core/src/server_ops.rs

```diff
@@ -31,10 +29,9 @@ fn message_into_diagnostic(msg: Message) -> lsp_types::Diagnostic {
         };
         lsp_types::Range { start, end }
     };
-    let code = Some(lsp_types::NumberOrString::String(msg.kind.body()));
```

Comment:
Thanks for catching this, I'm pretty sure it's a regression issue from when the code interface changed